packages/cozy-client/src/associations/HasManyFiles.js

```diff
@@ -7,7 +7,7 @@ import { CozyClientDocument } from '../types'
 import { DOCTYPE_FILES } from '../const'
 import Association from './Association'
 import HasMany from './HasMany'
-import { CouchDBViewCursor, DocId, ViewKey } from '../types'
+import { CouchDBViewCursor, DocId, ViewKey, IOCozyFile } from '../types'
```

Comment:
If this is just for the jsdoc, you should not import it that way but rather something like: 
```
@param  {import('../types').IOCozyFile} file
```

Reply:
Why is it better? Does it allow to make the import only if the function is actually called from the app?

Reply:
because `import` is used by the transpiler / bundler. If you import it only for documentation, you import it. And the file will be available in the bundle. If you /*@param import('')*/ then the file will not be available in the bundle.

Reply:
Ok nice ; but don't we exclusively use the types for documentation, i.e. in the JSDocs ? Is there legitimate use for importing types at the beginning of the file ?

Reply:
No.